src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -486,15 +488,39 @@ protected function backfill_multiday_event_ids( array $events = [] ) {
 				continue;
 			}
 
+			$site_timezone     = Timezones::build_timezone_object();
+			$utc               = Timezones::build_timezone_object( 'UTC' );
+			$use_site_timezone = Timezones::is_mode( 'site' );
+
 			/** @var \DateTime $event_day */
 			foreach ( $event_period as $event_day ) {
 				$event_day_string = $event_day->format( Dates::DBDATEFORMAT );
+				$start            = tribe_beginning_of_day( $event_day->format( Dates::DBDATETIMEFORMAT ) );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 108 characters

<sub>hash: be477df469f121e2626ca983a23e5503</sub>